### PR TITLE
Update TRIVIA.md

### PR DESCRIPTION
**Pull request by ben45c**

### CODE REVIEW

**Comment by ben45c** [comment]
Signed.

**Comment by fitojb** [comment]
It would be great to have a more descriptive commit message than GitHub’s default of “Update <filename>”. The log is already hard to parse due to this.

**Comment by eliheuer** [comment]
I like the spirit of this PR, thank you!

There is a need for an index of all know libre fonts. I'm in the planning stages of building something like this at libretype.org. It wouldn't be a site that hosts libre fonts and displays specimens such as what Google Fonts is doing, but would simply be an index of what exists in the libre type commons.
